Sources/ProjectSpec/SpecFile.swift

```diff
@@ -13,17 +13,21 @@ public struct SpecFile {
     fileprivate struct Include {
         let path: Path
         let relativePaths: Bool
+        let optional: Bool
 
         static let defaultRelativePaths = true
+        static let defaultOptional = false
 
         init?(any: Any) {
             if let string = any as? String {
                 path = Path(string)
                 relativePaths = Include.defaultRelativePaths
+                optional = Self.defaultOptional
```

Comment:
`Self.` will work only on Swift 5.1 or above.

Reply:
I've updated the rest of the codebase to Swift 5.1 in https://github.com/yonaskolb/XcodeGen/pull/714

Reply:
I'll review #714